projects/elife.yaml

```diff
@@ -143,13 +157,32 @@ defaults:
         cpucap: 100 # percent (vagrant default)
 
 basebox:
+    formula-repo: https://github.com/elifesciences/basebox-formula
     aws:
         ec2:
             ami: ami-9eaa1cf6 # Ubuntu 14.04 (correct, but older)
         ports:
             - 22
     vagrant: {}
 
+heavybox:
+    description: a project-less box with a bunch of features turned on
+    formula-repo: https://github.com/elifesciences/basebox-formula
+    aws:
+        ec2:
+            ami: ami-1d4e7a66 # xenial, us-east-1, build date 20170811, hvm:ebs-ssd
+            masterless: true
+        rds:
+            storage: 5
+            engine: MySQL
+            version: 5.7
+            
```

Comment:
if you want to go the extra mile, this could have a small `ext` so that we test `elife.external-volume` in builder-base-formula thoroughly